lib/em-hiredis/redis_client.rb

```diff
@@ -26,7 +26,8 @@ def initialize(
         uri,
         inactivity_trigger_secs = nil,
         inactivity_response_timeout = nil,
-        em = EventMachine)
+        em = EventMachine,
+        options = {})
```

Comment:
This new parameter should be documented alongside the others.

Reply:
Do you mean `reconnect_attempts` should go along with `inactivity_trigger_secs` and `inactivity_response_timeout`? If so, wouldn't this be a breaking change then? since we don't use keyword parameters

Reply:
oh, I see what you mean 🤦🏻‍♀️ I will document them alongside the others